pkg/rulefmt/rulefmt.go

```diff
@@ -137,9 +137,11 @@ func (r *Rule) Validate() (errs []error) {
 		}
 	}
 
-	for k := range r.Annotations {
-		if !model.LabelName(k).IsValid() {
-			errs = append(errs, errors.Errorf("invalid annotation name: %s", k))
+	if r.Alert != "" {
```

Comment:
this isn't really needed since L119 checks that recording rules have 0 annotations.

Reply:
yes. thanks